src/app/components/date-range-select/date-range-select.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { DateAdapter } from '@angular/material/core';
+import { Moment } from 'moment';
+import { DateSelectService } from 'src/app/services/date-select.service';
+
+@Component({
+  selector: 'app-date-range-select',
+  templateUrl: './date-range-select.component.html',
+  styleUrls: ['./date-range-select.component.scss']
+})
+export class DateRangeSelectComponent implements OnInit {
+  startDateCtrl = new FormControl(new Date());
+  endDateCtrl = new FormControl(new Date());
+  minDate = new Date(2020, 0, 1);
+  maxDate = new Date(Date.now());
```

Comment:
Ah true about the `newDate()`!

Otherwise, the `minDate` and `maxDate` values here are used to specify the earliest possible dates that can be picked on the picker. So if you scroll all the months in the calendar popup thing back, you can't go further back than Jan 2020, or further forward than today (we will want to change this once we are predicting data).

The 30/7/2020-1/9/2020 dates that come up by default are retrieved from date service, since it starts up with a default range of `3 days ago` to `yesterday`.

Reply:
Looks more like 2 months ago to last month